Plotly.Blazor/Plotly.Blazor.csproj

```diff
@@ -31,6 +31,7 @@ plotly.js is free and open source and you can view the source, report issues or
     <PackageReference Include="Microsoft.AspNetCore.Components" Version="3.1.8" />
     <PackageReference Include="Microsoft.AspNetCore.Components.Web" Version="3.1.8" />
     <PackageReference Include="Microsoft.CSharp" Version="4.7.0" />
+    <PackageReference Include="Newtonsoft.Json" Version="12.0.3" />
```

Comment:
Is there an alternative for using this library? We have intended to remain independent from Newtonsoft.